canonical/juju-dashboard
# WD-9042 - Validate secrets

## Done

- Validate secrets in the config panel.

## QA

- Open the config panel.
- Enter a secret URI that doesn't exist.
- A validation error should appear.
- Enter a valid URI.
- Then error shouldn't appear.

## Details

- https://warthogs.atlassian.net/browse/WD-9042
